### PR TITLE
add hera Elementary-OS 5.1

### PR DESCRIPTION
**Pull request by IslamAlam**

### CODE REVIEW

**Review by tprasadtp — CHANGES_REQUESTED** [review]
Could you also update the table at `docs/faq/distros.md` to include Elementary 5.1,
Other than that it looks fine.

File: after-effects [hunk 1]
```diff
@@ -650,9 +650,9 @@ function _fix_ubuntu_derivatives()
                                             code_name="xenial"
                                             _log_success "${_elementary} Changed codename to $code_name"
                                             ;;
-      juno )                                _log_and_show "${_elementary} This release of Elementary OS is based on Ubuntu 18.04 Bionic."
+      juno | hera)                          _log_and_show "${_elementary} This release of Elementary OS is based on Ubuntu 18.04 Bionic."
```
Comment (tprasadtp) [inline after-effects:653]: Is the release stable? If not a warning should be added.

Comment (IslamAlam) [reply after-effects:653]: I have updated the table at `docs/faq/distros.md`.
The stable release of 5.1 "Hera"  was released on 3 December 2019.

**Review by IslamAlam — COMMENTED** [review]
I have updated the table at `docs/faq/distros.md.`
The stable release of 5.1 "Hera" was released on 3 December 2019.

**Review by tprasadtp — APPROVED** [review]
Perfect! Thank you for your contribution!

**Pull request merged**